client/views/list/list.js

```diff
@@ -283,9 +288,38 @@ Template.list.helpers({
         tableMod = true;
       }
     }
-
     return tableAdmin || tableMod;
   },
+  hiddenQuestionsPresent() {
+    let questions;
+    if (Session.get('search') === 'all') {
+      questions = Template.instance().visibleQuestions.find({
+        instanceid: Template.instance().data._id,
+      }).fetch();
+    } else {
+      const re = new RegExp(Session.get('search'), 'i');
+      questions = Template.instance().visibleQuestions.find({
+        instanceid: Template.instance().data._id,
+        $or: [{
+          text: {
+            $regex: re,
+          },
+        }, {
+          poster: {
+            $regex: re,
+          },
+        }],
+      }).fetch();
+    }
+    let noOfHiddenQues = 0;
+    questions.forEach(function(question) {
+      if (question.state === 'disabled')
+        noOfHiddenQues += 1;
+    });
+    if(noOfHiddenQues > 0)
```

Comment:
You could return true from inside the function when you get the first hidden question.
That would make it more efficient and faster as in that case comparatively lesser number of iterations would be required.